x/poolmanager/create_pool.go

```diff
@@ -52,19 +45,22 @@ func (k Keeper) CreatePool(ctx sdk.Context, msg types.CreatePoolMsg) (uint64, er
 		return 0, err
 	}
 
+	// createPoolZeroLiquidityNoCreationFee contains shared pool creation logic between this function (CreatePool) and
+	// CreateConcentratedPoolAsPoolManager. Despite the name, within this (CreatePool) function, we do charge a creation
+	// fee and send initial liquidity to the pool's address. This function is strictly used to reduce code duplication.
```

Comment:
unclear what is referred to as this
```suggestion
	// CreateConcentratedPoolAsPoolManager. Despite the name, within this (CreatePool) function, we do charge a creation
	// fee and send initial liquidity to the pool's address. createPoolZeroLiquidityNoCreationFee is strictly used to reduce code duplication.
```